projects/lendai-us/design-system/src/lib/layout/landing-page-layout/landing-layout.service.ts

```diff
@@ -0,0 +1,21 @@
+import { InjectionToken } from '@angular/core';
+import { Observable } from 'rxjs';
+import { IconGlyph } from '../../icon';
+
+export interface LandingLayoutCategory {
```

Comment:
i think we can create a shared interface for navigation links (call it something like `LayoutCategory`) and re-use it in both `LanginLayout` module and in `SidenavLayout` module